test/GlobalFunctions/CanDeclareGlobalFunctionNonEval.baseline

```diff
@@ -0,0 +1,2 @@
+TypeError: Redeclaring a global function that is not configurable, not writable, and not enumerable is not allowed.
+	at Global code (CanDeclareGlobalFunctionNonEval.js:1:1)

```

Comment:
Any chance we can get the name of the function to appear in the error message, or is that information not convenient to plumb through?

Reply:
Looks like there's no perrors.h cases with %s so I'd assume that's not supported and maybe the rterrors.h codes can't be used in this context?

I'm just thinking it would be a much better dev experience if the message they got said "Redeclaring global property 'undefined'" or some such. On that note, undefined is not actually a function, so is this error message a bit misleading? Does the clause generally apply to any property, function or otherwise, with the descriptor restrictions "not configurable, not writable, and not enumerable"?